runtime/runtime_impl_jetty12/src/main/java/com/google/apphosting/runtime/jetty/JettyServletEngineAdapter.java

```diff
@@ -138,10 +137,12 @@ public void run(Runnable runnable) {
       httpConfiguration.setSendDateHeader(false);
       httpConfiguration.setSendServerVersion(false);
       httpConfiguration.setSendXPoweredBy(false);
+      httpConfiguration.setUriCompliance(UriCompliance.LEGACY);
```

Comment:
I would not set this all the time.   We want new behaviour for >= EE10 deployments

Reply:
Now only setting this if `LEGACY` mode is set or we are configured to use EE8.